### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by urbantumbleweed**
#### What's this PR do?

This adds tests to frylord.  It covers utils, reducers, and methods
#### What are the important parts of the code?

go to src/_module_/**test** for each of the modules listed above and see the tests written.
#### How should this be tested by the reviewer?

download the branch, `npm install && npm run test`

### CODE REVIEW

File: src/methods/__tests__/write-file.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const dir = path.dirname(filepath);
+  const base = path.basename(filepath);
+  const data = 'this is sample text';
+
+  it('creates a file in a given directory provided in filepath', function(done){
+    writeFile(filepath, data)
+      .then(function(entryArr){
+        expect(entryArr[1].fullPath).toEqual(dir);
+      })
+      .then(() => rm('this'))
```
Comment (phated) [inline src/methods/__tests__/write-file.js]: prefix with / (/this)

File: src/methods/__tests__/write-file.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      .then(() => done(), done);
+  });
+
+  it('creates a file with a name provided in filepath', function(done){
+    writeFile(filepath, data)
+      .then(function(entryArr){
+        expect(entryArr[2][0].name).toEqual(base);
+      })
+      .then(() => rm('this'))
```
Comment (phated) [inline src/methods/__tests__/write-file.js]: prefix with / (/this)

File: src/methods/__tests__/write-file.js (hunk 1 above)
Comment (phated) [inline src/methods/__tests__/write-file.js]: these should probably just be an afterEach

File: src/reducers/__tests__/content.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    const state = content(initial, action);
+    expect(state).toEqual('');
+    done();
+  });
+
+  it('returns the initial state if action type does not match', function(done){
+    const initial = 'test';
+    const action = {};
+    const state = content(initial, action);
+    expect(state).toEqual(state);
```
Comment (phated) [inline src/reducers/__tests__/content.js]: state toEqual initial

File: src/reducers/__tests__/directory.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+const directory = require('../directory');
+const {
+  SAVE_FILE,
+  DELETE_FILE,
+  CHANGE_DIRECTORY,
+  REFRESH_DIRECTORY
+} = require('../../constants');
+
+describe('#directory', function(){
```
Comment (phated) [inline src/reducers/__tests__/directory.js]: directory reducer

File: src/reducers/__tests__/filename.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      const initial = undefined;
+      const type = UPDATE_FILENAME;
+      const state = filename(initial, { type, payload });
+      expect(state).toEqual(payload.filename);
+      done();
+    });
+  });
+
+  it('returns undefined when type matches and action.payload.filename is undefined', function(done){
+      const initial = 'initial';
```
Comment (phated) [inline src/reducers/__tests__/filename.js]: indent is messed up here

File: src/reducers/__tests__/notification.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var expect = require('expect');
```
Comment (phated) [inline src/reducers/__tests__/notification.js]: using var

File: src/reducers/__tests__/notification.js [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+var notification = require('../notification');
```
Comment (phated) [inline src/reducers/__tests__/notification.js]: using var

File: src/reducers/__tests__/notification.js [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+  SAVE_FILE,
+  DELETE_FILE,
+  CHANGE_FILE,
+  DELETE_DIRECTORY,
+  CHANGE_DIRECTORY
+} = require('../../constants');
+
+describe('#notifcation', function(){
+  const payload = {
+      notification: 'some notification'
```
Comment (phated) [inline src/reducers/__tests__/notification.js]: messed up indent

File: src/reducers/__tests__/filename.js [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+const expect = require('expect');
+
+const filename = require('../filename');
+const {
+  CHANGE_FILE,
+  UPDATE_FILENAME
+} = require('../../constants');
+
+describe('#filename', function(){
```
Comment (phated) [inline src/reducers/__tests__/filename.js]: filename reducer

File: src/reducers/__tests__/notification.js [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+const {
+  ERROR,
+  SAVE_FILE,
+  DELETE_FILE,
+  CHANGE_FILE,
+  DELETE_DIRECTORY,
+  CHANGE_DIRECTORY
+} = require('../../constants');
+
+describe('#notifcation', function(){
```
Comment (phated) [inline src/reducers/__tests__/notification.js]: notification reducer

File: src/reducers/__tests__/projects.js [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+const expect = require('expect');
+
+const projects = require('../projects');
+
+const {
+  DELETE_DIRECTORY,
+  CHANGE_DIRECTORY
+} = require('../../constants');
+
+describe('#projects', function(){
```
Comment (phated) [inline src/reducers/__tests__/projects.js]: projects reducer

File: src/reducers/__tests__/status.js [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+const {
+  ERROR,
+  SAVE_FILE,
+  DELETE_FILE,
+  CHANGE_FILE,
+  DELETE_DIRECTORY,
+  CHANGE_DIRECTORY
+} = require('../../constants');
+
+describe('#status', function(){
```
Comment (phated) [inline src/reducers/__tests__/status.js]: status reducer

File: src/utils/__tests__/refreshDirectoryAction.js [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+const expect = require('expect');
+const { isFSA } = require('flux-standard-action');
+
+const refreshDirectoryAction = require('../refresh-directory-action');
+const { REFRESH_DIRECTORY } = require('../../constants');
+
+describe('refreshDirectoryAction Util', function(){
+  const ref = {
+        listing: 'item one',
```
Comment (phated) [inline src/utils/__tests__/refreshDirectoryAction.js]: bad indent

**Comment by urbantumbleweed** [comment]
I have updated this this repo and addressed the issues pointed out.  The linter was a big help.

**Pull request merged**